spec/activejob/queue_adapters/dispatch_rider_adapter_spec.rb

```diff
@@ -43,7 +43,7 @@ def perform(*)
       end
 
       it("is called when performing later") {
-        expect(adapter).to receive(:enqueue).with instance_of(foo_job_class)
+        #expect(adapter).to receive(:enqueue).with instance_of(foo_job_class)
```

Comment:
@mathieujobin beacuse of the changes you made in https://github.com/payrollhero/activejob-dispatch_rider/pull/6/commits/9be6554e735a90bed3da645ff26495395f57ab9e, where enqueue is no longer a classmethod, but a regular method, so it should be `expect_any_instance_of(adapter)`.